apps/src/constants.js

```diff
@@ -58,6 +58,7 @@ exports.TestResults = {
   TOO_MANY_BLOCKS_FAIL: 20,               // More than the ideal number of blocks were used.
   APP_SPECIFIC_ACCEPTABLE_FAIL: 21,       // Application-specific acceptable failure.
   MISSING_RECOMMENDED_BLOCK_FINISHED: 22, // The level was solved without a recommended block
+  PASS_WITH_EXTRA_TOP_BLOCKS: 23,         // There was more than one top-level block.
```

Comment:
Whoops!  I was looking at [the change here](https://github.com/code-dot-org/code-dot-org/pull/6875/commits/23746f4d97dc0150cd3bfac61d8deb8698bd2628#diff-b27beea3a767adec7120b1cc4263f9bbR33) but it looks like you undid it on purpose since this feature is behind an experiment.  My bad!